deploy/playbooks/roles/common/templates/prod_db.py.j2

```diff
@@ -1,7 +1,15 @@
 # {{ ansible_managed }}
 
-sqlalchemy = {
-    'url': 'postgresql+psycopg2://{{ app_name }}:{{ db_password.stdout }}@127.0.0.1/{{ app_name }}',
+sqlalchemy_w = {
+    'url': 'postgresql+psycopg2://{{ app_name }}:{{ postgresql_app_user_password }}@{{ master_ip }}/{{ app_name }}',
```

Comment:
true. I decided to leave that to a separate PR so that I could have something that already works